### ISSUE TITLE
on windows 7: TemplateNotFound: tiles\openstreetmap\tiles.txt

### ISSUE THREAD

**Issue by fivejjs** [python-visualization/folium#129]
I am trying:
```
map_osm = folium.Map(location=[151.179664, -33.932643])
```

**Comment by fivejjs** [comment]
Modify code as below can make jinja2 find the templates:
line 165 of folium.py change to:
```
for tile in self.default_tiles:
            # join_tile_path = functools.partial(os.path.join, 'tiles', tile)
            # patch to fit code with windows
            join_tile_path = ['tiles', tile]
            self.tile_types[tile] = {
                'templ': self.env.get_template('/'.join(join_tile_path+['tiles.txt'])),
                'attr': self.env.get_template('/'.join(join_tile_path+['attr.txt'])),
            }
```

**Comment by ocefpaf** [comment]
What version of folium are you using?  This was fixed in #63 some time ago.  I am closing this for now.  Feel free to re-open it if you are experiencing this with the latest release.

Thanks.
